test/box-luatest/gh_9923_invalid_iproto_index_name_mapping_on_the_server_side_test.lua

```diff
@@ -0,0 +1,88 @@
+local net = require('net.box')
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new({alias = 'master'})
+    cg.server:start()
+    cg.server:exec(function()
+        box.schema.create_space('test', {
+            id = 621,
+            temporary = true,
+            if_not_exists = true,
```

Comment:
Since you create the space and indexes in `before_all`, you don't need the `if_not_exists` attribute. Let's drop all `if_not_exists` attributes.

Reply:
fix